src/features/cardView/PokemonCardView/PokemonCardView.tsx

```diff
@@ -0,0 +1,55 @@
+import { IPokemonData } from "../../../types/pokemonTypes";
+import PokemonBasicCard from "../PokemonBasicCard/PokemonBasicCard";
+import strength from "../../../assets/icons/strength.svg";
+import { CardViewContainer } from "./styles";
+import { useState } from "react";
+import Modal from "../../../components/Modal/Modal";
+import PokemonModalCard from "../../PokemonModalCard/PokemonModalCard";
+import { transformPokemonDataToAttributes } from "../../../utils/transformData";
+
+interface PokemonCardViewProps {
+  data: IPokemonData[];
+}
+
+const PokemonCardView = ({ data }: PokemonCardViewProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
I think you don't need this state
the modal is open when selectedCard is not null, right?